cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -49,9 +49,7 @@ most likely need other command line flags. For more information you can invoke
 
 ## Configuring node group auto discovery
 
-If you do not configure node group auto discovery, cluster autoscaler will attempt
-to match nodes against any scalable resources found in any namespace and belonging
-to any Cluster.
+You must configure node group auto discovery to inform cluster autoscaler which cluster in which to find for scalable node groups.
```

Comment:
@elmiko FYI, I think this original statement is not correct. If you don't provide `autoDiscovery` configuration via the helm chart, you get an error.

Reply:
looking at this https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller.go#L419 it leads me to believe that if unconfigured then it should be watching everything. but perhaps the permissions for the rbac are not allowing it?

Reply:
I haven't attempted to get further beyond the helm chart validation, so I haven't actually tried to get it to work using the runtime.

But because we can only pass in one kubeconfig, and that kubeconfig is only going to work with a single cluster (thinking of `kubeconfig-incluster` context here, so CA is installed on the mgmt cluster and managing a remote workload cluster) why would you pass in a specific workload cluster kubeconfig secret but not declare the specific workload cluster that that secret pairs with?

Maybe this scenario makes sense for a self-managed CAPI cluster? (CAPI control plane installed onto the workload cluster)

Reply:
> why would you pass in a specific workload cluster kubeconfig secret but not declare the specific workload cluster that that secret pairs with?

+1, seems like it should be specified explicitly